testing/acme.sh/APKBUILD

```diff
@@ -2,23 +2,21 @@
 # Maintainer: Francesco Colista <fcolista@alpinelinux.org>
 pkgname=acme.sh
 pkgver=2.8.0
-pkgrel=0
+pkgrel=1
 pkgdesc="An ACME Shell script, an acme client alternative to certbot"
 url="https://github.com/Neilpang/acme.sh"
 arch="noarch"
 license="GPL-3.0"
-depends="curl"
+depends="curl cmd:openssl"
 source="$pkgname-$pkgver.tar.gz::https://github.com/Neilpang/acme.sh/archive/$pkgver.tar.gz"
 builddir="$srcdir/$pkgname-$pkgver"
 
 build() {
 	cd "$builddir"
-	return 0
 }
 
 check() {
 	cd "$builddir"
-	return 0
 }
 
```

Comment:
You can remove both the build and check function since they do nothing.

Reply:
ah, ok. Wasn't sure if they were needed or not

Reply:
abuild defines all these functions. When you put them in in your APKBUILD, you just override them.